Documentation/O3DEMPSProjectExportTesting.md

```diff
@@ -0,0 +1,58 @@
+# O3DE MPS Project Export Testing Instructions
+
+## Notes
+
+* The general O3DE project-centric export script is still in development. For now the project-centric instructions will be calibrated using the MPS Gamelift Export Script. Once the general script is merged, the instructions will be updated.
+* This instruction set assumes the host platform is Windows. Instructions for Linux coming shortly.
+
+## Fresh Install and Setup
+1. Clone the development branch of O3DE:
+```bash
+cd \path\to\workspace
+git clone https://github.com/o3de/o3de.git # this should already be in development branch
+```
+2. Clone the development branch of O3DE MPS:
+```bash
+git clone https://github.com/o3de/o3de-multiplayersample.git # this should already be in development branch
+```
+3. Clone O3DE MPS Assets, then switch to development branch. Afterwards initialize any submodules (such as O3DEPopcornFX), and make sure they are on development:
+```bash
+git clone https://github.com/o3de/o3de-multiplayersample-assets.git
+cd o3de-multiplayersample-assets
+git switch development
+git lfs pull
+git submodule update --init --recursive
+cd Gems/O3DEPopcornFXPlugin
+git switch development
```

Comment:
We should go with what -assets repro pointing to, instead of modifying the submodule in testing.

Reply:
gotcha, so if issues come up related to the wrong commit being pointed to, that is something we have to fix. Makes sense.